.github/workflows/Pre_release.yml

```
@@ -12,29 +20,25 @@ jobs:
   release:
     name: "Publish Pre-Release"
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
     steps:
       - uses: actions/checkout@v4
         with:
-          fetch-depth: 0  # Needed determine latest tag
+          fetch-depth: 0  # Needed determine tags
           persist-credentials: true
```

Could be part of your problem:
```suggestion
          persist-credentials: false  # Avoid using these credentials as pushing new commits doesn't re-trigger CI jobs
```